### PR TITLE
Fixed grammar and spelling errors

### PR DESCRIPTION
**Pull request by TiredMir**
Will update in future.

### CODE REVIEW

File: book_vol_1.txt [hunk 1]
```diff
@@ -721,35 +721,35 @@ ESP -> |SAVED EPB | --> Save the previous stack frame
...
        ESP = 7fffff4C
 
-       The only thing that sub() does is returning 0xBEEF. As it was mentioned before, EAX register is mostly used for functions' return value. After
-executing the function's prologue (PUSH EBP and MOV EBP,ESP), the hex value 0xBEEF is gonna be put in EAX. Here's how the stack will look like:
+  The only thing that sub() does is returning 0xBEEF. As it was mentioned before, the EAX register is mostly used for the function's return value.
+  after executing the function's prologue(PUSH EBP and MOV EBP,ESP), the hex value 0xBEEF is gonna be put in EAX. Here's how the stack will look like:
 
         +----------+
-        |Saved EIP | --> Return to whoever called main()
+        |Saved EIP | --> Return to whichever address called main()
```
Comment (Captainarash) [inline book_vol_1.txt]: whoever should be intact :)

File: book_vol_1.txt [hunk 2]
```diff
@@ -69,8 +69,8 @@ who is interested can learn and contribute.
 
 About the Author[s]:
 
-        Arash TC is the main author and maintainer of this book. He is currently studying IT in Finland. He's got OSCP/OSCE and a long long long way
-ahead to become a pro. He will appreciate reader's comments and criticisms and contributions. His main interest is low level security and kernel
+        Arash TC is the main author and maintainer of this book. He is currently studying IT in Finland. He's got an OSCP/OSCE and a long long long way
```
Comment (Captainarash) [inline book_vol_1.txt]: OSCP and OSCE are 2 separate certs

File: book_vol_1.txt [hunk 3]
```diff
@@ -80,7 +80,7 @@ internals.
 Introduction:
 
         This book/guide/tutorial/wiki is about assembly and x86 architecture. It's written by a low level security dude for low level security dudes.
-If you wanna learn Assembly and its structure, reversing basics, Segmentation, Paging and etc. keep on reading. I highly recommend you check
+If you wanna learn Assembly and its structure, reversing basics, segmentation, Paging, etc. keep on reading. I highly recommend you check
```
Comment (Captainarash) [inline book_vol_1.txt]: Segmentation is the tile so I wrote it Uppercase.

File: book_vol_1.txt [hunk 4]
```diff
@@ -97,17 +97,17 @@ Since the book is not finished yet, it's not so convenient to put a table of con
 Data Types:
 
         There are 5 different data types to deal with in the world of Assembly. They are as follows:
-        Byte: A byte is simply an 8-bit value (1 byte) and the C equivalent of a Byte is when you define a character like:
+        Byte: A byte is simply an 8-bit value (1 byte) and the C equivalent of a byte is when you define a character like:
```
Comment (Captainarash) [inline book_vol_1.txt]: Byte, Word, Double-Word .... are emphasized using uppercase

**Review by Captainarash — CHANGES_REQUESTED** [review]

File: book_vol_1.txt (hunk 2 above)
Comment (TiredMir) [reply book_vol_1.txt]: Yeah I know but there was a slash so I thought you merged them together, I'll fix it with some other stuff and make a pull request tonight...

File: book_vol_1.txt (hunk 4 above)
Comment (TiredMir) [reply book_vol_1.txt]: A byte usually isn't in uppercase by convention... but I'll change it.

File: book_vol_1.txt (hunk 1 above)
Comment (TiredMir) [reply book_vol_1.txt]: Ok

**Comment by TiredMir** [comment]
Changes submitted

**Review by Captainarash — APPROVED** [review]
Thank you! Great Work.

**Pull request merged**